apps/expo/tailwind.config.ts

```diff
@@ -2,9 +2,50 @@ import type { Config } from "tailwindcss";
 // @ts-expect-error - no types
 import nativewind from "nativewind/preset";
 
-import baseConfig from "@acme/tailwind-config";
```

Comment:
Why?

Reply:
Nativewind does not support some of the features we are using in the nextjs app, namely  ```  borderRadius: {
        lg: "var(--radius)",
        md: "calc(var(--radius) - 2px)",
        sm: "calc(var(--radius) - 4px)",
      }, ```

You can see more https://github.com/marklawlor/nativewind/issues/764 here

Reply:
Maybe split the preset up then to core/web/native?